spec/menu_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'menu'
+
+describe Menu do
+
+before(:each) do
+  subject.add_dish("Tomatoes mozarella",10)
+end
+
+  it 'allows to add dishes to the list with name and price' do
+    expect(subject).to respond_to(:add_dish).with(2).arguments
+  end
+
+  it 'allows to display dishes as a well formatted list' do
+    subject.add_dish("Prosciutto",12)
+    expect(subject.display_menu).to eq("Tomatoes mozarella: £10, Prosciutto: £12")
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)